data/config/datasets/adaguc.tests.solarterminator.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Can you add the new option to https://github.com/KNMI/adaguc-server/blob/master/doc/configuration/Layer.md ?

Can you also write a tutorial in https://github.com/KNMI/adaguc-server/blob/master/doc/tutorials/Readme.md? E.g. copy this dataset configuration to the right place. And then looking at it in the viewer?

Reply:
Very good idea! Regarding Layer.md, I should probably add it under LiveUpdate, right?

Reply:
OK, I have a not-so-good update. Basically the legend of the solar terminator is now broken. If I make the GetLegendGraphic call it works fine (I have added a test for that), but when running Adaguc locally and accessing the dataset from the viewer, no styles are listed. 
I think this is related to the metadata db setup (_readFromDb_ is set to false). Maybe related to the variable/dimension check? There is a comment saying _If variable name equals dimension name, values are stored in the database._  
Any clues how to fix this?

Reply:
![image](https://github.com/user-attachments/assets/804989b3-25f8-4b59-874c-05399cc0fd6e)

Reply:
In adagucserverEC/utils/XMLGenUtils.cpp, line 751, you need to add ||
      metadataLayer->dataSource->dLayerType == CConfigReaderLayerTypeLiveUpdate

Not sure what other consequences this has.

Reply:
Good find!